pdr_backend/analytics/predictoor_dashboard/dash_components/util.py

```diff
@@ -143,3 +160,30 @@ def select_or_clear_all_by_table(
         selected_rows = list(range(len(rows)))
 
     return selected_rows
+
+
+@enforce_types
+def process_user_payout_stats(
```

Comment:
Looks much cleaner now, but we should add a better name to this function: "get_predictoors_data_from_payouts" or something similar